.chloggen/docker-receiver.yaml

```diff
@@ -0,0 +1,28 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: dockerreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: feat: add a new dockerreceiver to replace dockerstatsreceiver
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [31597]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: Currently, the receiver does the same things as Docker stats receiver. |
+         In the future, the receiver will be able to fetch also Docker events and container logs via Docker API.
+
```

Comment:
Is changelog workflow failing because of incorrect usage of pipe here?

Reply:
Do it like this: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/1bb4967f33f26189e65a72f19b65fa50962ac698/.chloggen/add_serviceName_column_to_metrics_tables.yaml

```
subtext: |
  It's a breaking change. users who upgrade to the latest version need to alter the Clickhouse table:
  bla bla bla next line
  another line
```